vmm/src/config.rs

```diff
@@ -1930,9 +1930,18 @@ impl VmConfig {
             }
         }
 
+        // The 'conflict' check is introduced in commit 24438e0390d3
+        // (vm-virtio: Enable the vmm support for virtio-console).
+        //
+        // Allow double tty mode, for someone want to use virtio console
+        // for better performance, and want to keep legacy serial to catch
+        // boot stage logs for debug.
+        // Using such double tty mode, you need to configure the kernel
+        // properly, such as:
+        // "console=hvc0 earlyprintk=ttyS0"
         if self.console.mode == ConsoleOutputMode::Tty && self.serial.mode == ConsoleOutputMode::Tty
         {
-            return Err(ValidationError::DoubleTtyMode);
+            info!("Notice, {}", ValidationError::DoubleTtyMode);
```

Comment:
```suggestion
            info!("{}", ValidationError::DoubleTtyMode);
```

No other info message in Cloud Hypervisor is prefixed with "Notice".

Reply:
Thanks will update.